### PR TITLE
Release 1.5.0

### PR DESCRIPTION
**Pull request by WackerO**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/differentialabundance/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/differentialabundance _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: .github/workflows/awsfulltest.yml [hunk 1]
```diff
@@ -15,6 +15,9 @@ jobs:
     steps:
       - name: Launch workflow via tower
         uses: seqeralabs/action-tower-launch@v2
+        # TODO nf-core: You can customise AWS full pipeline tests as required
+        # Add full size test data (but still relatively small datasets for few samples)
+        # on the `test_full.config` test runs with only one set of parameters
```
Comment (maxulysse) [inline .github/workflows/awsfulltest.yml:20]: I'd say this has been done already?
```suggestion
```

File: conf/test.config [hunk 2]
```diff
@@ -47,5 +47,9 @@ params {
 
     // Activate GSEA
     gsea_run = true
-    gsea_gene_sets = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/mus_musculus/gene_set_analysis/mh.all.v2022.1.Mm.symbols.gmt'
+    gene_sets_files = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/mus_musculus/gene_set_analysis/mh.all.v2022.1.Mm.symbols.gmt'
+
+    // Report options
+    report_round_digits = 3
+    report_contributors = 'Jane Doe\nDirector of Institute of Microbiology\nUniversity of Smallville;John Smith\nPhD student\nInstitute of Microbiology\nUniversity of Smallville'
```
Comment (maxulysse) [inline conf/test.config:54]: From Smallville, it should be Lana Lang

File: docs/images/workflow.png
Comment (maxulysse) [inline docs/images/workflow.png:1]: No more white background?

File: main.nf [hunk 3]
```diff
@@ -13,63 +13,62 @@ nextflow.enable.dsl = 2
 
 /*
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    GENOME PARAMETER VALUES
+    IMPORT FUNCTIONS / MODULES / SUBWORKFLOWS / WORKFLOWS
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
-params.gtf = WorkflowMain.getGenomeAttribute(params, 'gtf')
+include { DIFFERENTIALABUNDANCE  } from './workflows/differentialabundance'
```
Comment (maxulysse) [inline main.nf:20]:
```suggestion
include { DIFFERENTIALABUNDANCE   } from './workflows/differentialabundance'
```

**Review by maxulysse — COMMENTED** [review]
Looking good, minor comments

File: docs/images/workflow.png
Comment (WackerO) [reply docs/images/workflow.png:1]: I dunno, I think that was changed in some template update 🤔

File: conf/test.config (hunk 2 above)
Comment (WackerO) [reply conf/test.config:54]: I don't like her 😜

File: .github/workflows/awsfulltest.yml (hunk 1 above)
Comment (WackerO) [reply .github/workflows/awsfulltest.yml:20]: I cannot apply your suggestion 😢 github gives an error and tells me to refresh which doesn't help. I'll wait for a second review and then do this in a PR if necessary

File: main.nf (hunk 3 above)
Comment (WackerO) [reply main.nf:20]: See above, cannot commit suggestion -.-

**Review by pinin4fjords — APPROVED** [review]
Nice, looking good- think I reviewed all the component PRs in some detail anyway.

Adding my approval so you don't need to wait later, but you'll have to PR Maxime's points to dev to get them in (the branch protections are what prevent you from applying them here).

**Review by maxulysse — APPROVED** [review]

**Comment by WackerO** [comment]
Many thanks you two!

**Pull request merged**